vm/isolated/isolated.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	USBDevNums    []string `json:"usb_device_num"` // /sys/bus/usb/devices/
 	StartupScript string   `json:"startup_script"` // script to execute after each startup
 	Pstore        bool     `json:"pstore"`         // use crashlogs from pstore
+	UseSystemCfg  bool     `json:"use_system_cfg"` // whether to allow system-wide SSH configuration
```

Comment:
I would add "ssh" into the name somehow, e.g. "use_system_ssh_cfg" or "system_ssh_cfg".